src/index.js

```diff
@@ -5,217 +5,233 @@ process.env.SENTRY_DSN =
   'https://3cfc427c2f18404da8db2ff632c29e83:bcc57d150cb7407983e78d995825a058@sentry.cozycloud.cc/72'
 
 const {
-  BaseKonnector,
-  requestFactory,
-  cozyClient,
+  CookieKonnector,
+  solveCaptcha,
   log,
   errors,
   scrape,
-  saveBills,
   hydrateAndFilter,
   addData
 } = require('cozy-konnector-libs')
-const path = require('path')
-const request = requestFactory({
-  // debug: true,
-  cheerio: false,
-  json: true,
-  jar: true,
-  headers: {
-    'User-Agent':
-      'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0'
-  }
-})
+let request
 const cheerio = require('cheerio')
 const moment = require('moment')
 
 const baseUrl = 'https://www.deezer.com'
 const PLAYLIST_DOCTYPE = 'io.cozy.deezer.playlists'
 
-module.exports = new BaseKonnector(start)
-
-async function start(fields) {
-  log('info', 'Authenticating ...')
-  await authenticate(fields.login, fields.password)
-  log('info', 'Successfully logged in')
-
-  log('info', 'Parsing bills')
-  const $ = await fetchBillsPage()
-  const bills = parseBills($)
-
-  await removeOldFiles(bills, fields)
+class DeezerKonnector extends CookieKonnector {
+  async fetch(fields) {
+    request = this.requestFactory({
+      // debug: true,
+      cheerio: false,
+      json: true,
+      jar: true,
```

Comment:
The jar option is not needed with CookieKonnector (if you use the method from the previous comment)